prisma-fmt/src/get_config.rs

```diff
@@ -20,29 +20,31 @@ struct GetConfigError {
     message: String,
 }
 
-pub(crate) fn get_config(params: &str) -> String {
+pub(crate) fn get_config(params: &str) -> Result<String, String> {
     let params: GetConfigParams = match serde_json::from_str(params) {
         Ok(params) => params,
         Err(serde_err) => {
-            return json!({
+            // TODO: this should be a panic imho
+            return Err(json!({
                 "error": {
                     "message": serde_err.to_string(),
                 }
             })
-            .to_string();
+            .to_string());
```

Comment:
If the input arguments aren't compatible with the expected JSON-encoded string, I think we should explicitly panic.